app/controllers/employment.controller.js

```diff
@@ -0,0 +1,93 @@
+const db = require('../models');
+
+const { Company, User } = db;
+
+// Retrieve all employees of a company
+
+exports.retrieveEmployees = (req, res, next) => {
+  const {
+    body: { id },
+  } = req;
+
+  Company.findById(id).then((company) => {
+    User.findAll({
+      where: {
+        companyId: company.id,
+      },
+    })
+      .then((users) => {
+        console.log({ users });
+        res.status(200).json(users);
```

Comment:
you're sending back all the users sensitive info here. complete of passwordHash and all the jazz. map to sessionObjects